src/v2/circuit/run.ts

```diff
@@ -65,8 +65,17 @@ export function AxiomCircuitRunner(halo2Wasm: Halo2Wasm, halo2LibWasm: Halo2LibW
 
   let dataQuery = getNewDataQuery();
 
-  async function buildFromString(code: string, inputs: string, cachedResults?: { [key: string]: string }) {
+  config = { ...config };
+
+  const clear = () => {
+    dataQuery = getNewDataQuery();
     halo2Wasm.clear();
+    halo2LibWasm.config();
+  }
+
+  async function buildFromString(code: string, inputs: string, cachedResults?: { [key: string]: string }) {
+
```

Comment:
nit: empty space after function declaration and before clear() on a lot of these functions